main

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env ruby
+require './app'
+app = App.new
+
+def main(app)
+  list_options
+  number = gets.chomp.to_i
+  execute_option(app, number)
+end
+
+def list_options
+  puts 'Please choose an option by entering a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+end
+
+def execute_option(app, number)
+  option = {
+    1 => :list_books,
+    2 => :list_persons,
+    3 => :create_person_option,
+    4 => :create_book_option,
+    5 => :create_rental,
+    6 => :get_rental_option,
+    7 => :exit_app
+  }
+  if option.key?(number)
+    send(option[number], app)
+  else
+    puts 'Enter valid number'
+    main(app)
+  end
+end
+
+def exit_app(*)
+  puts 'Thank you for using this App'
+end
+
+def create_person_option(app)
+  print 'Do you want to create student (1) or a teacher (2)? [Input the number]: '
+  person_code = gets.chomp.to_i
+  app.create_person(person_code)
+  main(app)
+end
+
+def create_book_option(app)
+  print 'Title: '
+  title = gets.chomp
+  print 'Author: '
+  author = gets.chomp
+  app.create_book(title, author)
+  main(app)
+end
+
+def get_rental_option(app)
+  print 'ID of person: '
+  id = gets.chomp.to_i
+  app.get_rental(id)
+  main(app)
+end
+
+def list_books(app)
+  app.list_all_books
+  main(app)
+end
+
+def list_persons(app)
+  app.list_all_people
+  main(app)
+end
+
+def create_rental(app)
+  app.create_rental
+  main(app)
+end
```

Comment:
- [ ] Please from the project requirement, all these methods should be in the `app.rb` file which serves as your console app entry-point, and this `main.rb` should only define the entry point by presenting the user with a list of options to perform, It lets users choose an option, if needed, ask for parameters for the option and have a way to quit the app like example code below. :relaxed: 
    
    ```ruby
    # main.rb
    def main
      app = App.new()
    
      response = nil
    
      # [OPTIONAL] You can learn  about the ruby lessons about basic sintax like loops: https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/lessons/basic_syntax.md  
      # *Present the user with a list of options to perform.
      puts "Welcome to School Library App!\n\n"
      while response != "7"
        puts "Please choose an option by enterin a number:"
        puts "1 - List all books"
        puts "2 - List all people"
        puts "3 - Create a person"
        puts "4 - Create a book"
        puts "5 - Create a rental"
        puts "6 - List all rentals for a given person id"
        puts "7 - Exit"
        # *Lets users choose an option.
        response = gets.chomp
    
        case response
        when "1"
          app.list_books()
        when "2"
          app.list_people()
        when "3"
          app.create_person()
        when "4"
          app.create_book()
        when "5"
          app.create_rental()
        when "6"
          app.list_rentals_for_person_id()
        when "7"
          # *Have a way to quit the app.
          puts "Thank you for using this app!"
        end
    
        puts "\n"
      end
    end
    
    # 'main' method should be invoked at the end of the main.rb file, hint in this lesson https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/console_apps.md#console-app-entry-point
    main()
    ```